Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -4503,6 +4503,17 @@ float RenderLayerCompositor::zoomedOutPageScaleFactor() const
     return page().zoomedOutPageScaleFactor();
 }
 
+FloatSize RenderLayerCompositor::viewSize() const
```

Comment:
Ok, now it is clear what a viewSize() is -- how about enclosingFrameViewSize()?

Reply:
Or enclosingFrameViewVisibleSize()